webapp-django/crashstats/signature/static/signature/js/signature_tab_summary.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tab for displaying signature summary.
+ * Does not have any panels.
+ * Does not have any controls.
+ *
+ * @extends {SignatureReport.Tab}
+ * @inheritdoc
+ */
+SignatureReport.SummaryTab = function(tabName) {
+
+    var config  = {
+        'panels': false,
+        'dataDisplayType': 'table',
+        'pagination': false
+    };
+
+    SignatureReport.Tab.call(this, tabName, config);
+
+};
+
+SignatureReport.SummaryTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
```

Comment:
Is it just me, or does this whole code look quite complex. That's a lot of object orientation, that makes my head spin, just to be able to get out that it's 3 key-value pairs of config data.